### DISCUSSION TITLE
https://angular.love/component-initialization-without-ngoninit-with-async-pipes-for-observables-and-ngonchanges

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#366] [General]
# https://angular.love/component-initialization-without-ngoninit-with-async-pipes-for-observables-and-ngonchanges

Angular.love - a place for all Angular enthusiasts created to inspire and educate.

https://angular.love/component-initialization-without-ngoninit-with-async-pipes-for-observables-and-ngonchanges

**Comment by JannickB92** [comment]
Great article, well explained. Although I do think that when using vm's, I might just use it as input for the component. Thanks!
